game-to-flatpak

```diff
@@ -23,17 +23,20 @@
 local fg = require "lib.flatpak-game"
 
 function usage()
-	print("game-to-flatpak [--network] [--bundle] [--keep-build-dir] [--keep-repo-dir] [FILE...]")
+	print("game-to-flatpak [--network] [--bundle] [--repo=DIR] [--keep-build-dir] [--keep-repo-dir] [FILE...]")
 end
 
 local file = nil
 local options = {}
+
 for k, v in ipairs(arg) do
 	if string.sub (v, 1, 2) == "--" then
 		if v == '--network' then
 			options.network = true
+        elseif string.sub(v, 1, 7) == '--repo=' then
```

Comment:
Indentation. This is at the same level as the `--` check within the for loop in the review tool...